cl/phase1/network/services/attestation_service.go

```diff
@@ -229,7 +229,7 @@ func (s *attestationService) ProcessMessage(ctx context.Context, subnet *uint64,
 		Pks:        [][]byte{pubKey[:]},
 		GossipData: att.GossipData,
 		F: func() {
-			err = s.committeeSubscribe.CheckAggregateAttestation(att.Attestation)
+			err = s.committeeSubscribe.AggregateAttestation(att.Attestation)
```

Comment:
can you check whether we need to aggregate **OUTSIDE** the `F` callback?

Reply:
ok updated